### PR TITLE
[CL-90] Port and host differ when using Docker Desktop, this needs to be documented

### PR DESCRIPTION
**Pull request by guarde**

### CODE REVIEW

File: README.md
Comment (stevelw) [inline README.md:1]: Don't mix your commits

File: README.md [hunk 1]
```diff
@@ -11,50 +11,84 @@
 
 ## Setting up for local development
 
-### Prerequisites
+> [!IMPORTANT]  
+> This README assumes you have Docker installed. If you do not, please read [this](https://docs.docker.com/engine/install/) manual, based on your operating system. It will cover setting up and testing your very own Docker instance (lucky you!)
 
-This README assumes you have Docker installed. If you do not, please read [this](https://docs.docker.com/engine/install/) manual, based on your operating system. It will cover setting up and testing your very own Docker instance (lucky you!)
+### Requirements
+Node: `7.0.1`
```
Comment (stevelw) [inline README.md]: This should be a Markdown list, and there should be an empty line after headings for MD compatibility.

File: README.md [hunk 2]
```diff
@@ -11,50 +11,84 @@
...
 
 #### Cloning and initial setup
 
-1. Clone the repository by executing `git clone https://github.com/clowder-cloud/backend.git`
-2. Enter the cloned repository locally, `cd backend`
-3. Install all dependencies, `npm i`
+1. Clone the repository with
+```bash
+git clone https://github.com/clowder-cloud/backend.git
+```
```
Comment (stevelw) [inline README.md]: Contents of list items should be indented by 4 spaces or one tab

File: README.md [hunk 3]
```diff
@@ -11,50 +11,84 @@
...
-   1. The `DATABASE_URL` will be constructed of values from `docker-compose.local.yaml` and the output of previous steps.
-      1. If you have not changed any values, you can copy & paste this into your `.env` - `DATABASE_URL="postgresql://root:root@HOST:5432/clowder"` and skip to step 5
-      2. `POSTGRES_USER` will be our `USERNAME`
-      3. `POSTGRES_PASSWORD` will be our `PASSWORD`
-      4. `POSTGRES_DB` will be our `DATABASE`
-      5. The IP copied from [Setting up the database](###-Setting-up-the-database) will be our `HOST`
-
-#### Migration and Seeding
+2. Refer to the `example.env` for what KEY:VALUE pairs you need
+    1. For `DATABASE_URL`, the `DATABASE`, `USERNAME` and `PASSWORD` can be found in `docker-compose.local.yaml`. The `HOST` will be the result of `npm run get-docker-ip`
```
Comment (stevelw) [inline README.md]: There's only one item so this should be a paragraph not a list item.

**Review by stevelw — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (guarde) [reply README.md]: This is for future proofing - in case we plan to add anything else to the .env

**Comment by stevelw** [comment]
You've added these on the end again, so now you have circular PR dependencies. Hold tight, I'm going to fix your PRs with https://git-scm.com/docs/git-revert.html

**Review by stevelw — APPROVED** [review]

**Comment by stevelw** [comment]
@jeepies I can't select you to request a review, but does it let you review it?

You may want to create another branch to save the unrelated changes I undid to clean up this PR.

**Review by PhilTBatt — APPROVED** [review]

**Pull request merged**